sopel/modules/etymology.py

```diff
@@ -79,8 +81,10 @@ def f_etymology(bot, trigger):
         msg = "Can't connect to etymonline.com (%s)" % (ETYURI % web.quote(word))
         bot.say(msg, trigger.sender)
         return NOLIMIT
-    except (AttributeError, TypeError):
+    except AttributeError:
         result = None
+    except TypeError:
+        result = "No word to look for."
```

Comment:
Can you help me understand why you split apart this exception handling? `etymology(word)` will never explicitly raise `TypeError`, so I don't think we can equate that type (puns!) of exception to any specific error state.

Reply:
Hm... I overlooked that, but I think it's because of that:

```
>>> len(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```

So I think the solution is to not use `if len(word) == 0:`, but instead `if not word:` which will test for None, 0, False, and empty string, all without raising this `TypeError`.

Reply:
The len(word) == 0 was for .ety with a space. The TypeError is what you get when use .ety without anything else.

But I think that we don't care about it anymore. I made the changes that @Exirel said and added the blank line on the headers. Also, I think I squash the commits, but I'm not sure about it because it is my first time doing it.

I like the project so, maybe I was a bit lost like this past days, but I will work on it. :)

Reply:
Squash is 👌!

Still moves the blank line from inside to outside the module docstring header, though. Other than that (we try to avoid touching random whitespace in unrelated commits/PRs), ready to 🚢.